site/viamguides/src/plant-watering-device-workshop/plant-watering-device-workshop.md

```diff
@@ -46,50 +41,103 @@ Follow this workshop to learn how to set up an automatic plant watering system b
+1. Remove the microSD card from your computer when the installation is complete.
+
+### Connect with SSH
+
+1. Place the microSD card into your Raspberry Pi and boot the Pi by plugging it in to an outlet. A red LED will turn on to indicate that the Pi is connected to power.
+   > aside negative
+   > Make sure you are using a 5V 3A USB-C power supply to power your Raspberry Pi 4. Using a power supply with inadequate amperage can lead to instability, throttling, or unexpected behavior. Additionally, while USB boot is enabled by default on newer Raspberry Pi 4 models, older versions may require a firmware update to enable it. Refer to the [Raspberry Pi documentation](https://www.raspberrypi.com/documentation/computers/raspberry-pi.html) for detailed setup instructions and compatibility.
+1. Once the Pi is started, connect to it with SSH. From a command line terminal window, enter the following command. The text in <> should be replaced (including the < and > symbols themselves) with the username and hostname you configured when you set up your Pi.
+   ```bash
+   ssh <USERNAME>@<HOSTNAME>.local
+
+   # for example, my command would look like this:
+   ssh atacke@echo.local
+   ```
+1. If you are prompted “Are you sure you want to continue connecting?”, type “yes” and hit enter. Then, enter the password for your username. You should be greeted by a login message and a command prompt.
+   ![raspberry pi SSH login success](assets/sshLoginSuccess.png)
+1. Update your Raspberry Pi to ensure all the latest packages are installed
+   ```bash
+   sudo apt update
+   sudo apt upgrade
+   ```
 
-## Set up your Plant Watering Device
 
-Duration: 10
+## Set up your Plant Watering Device
+Duration: 15
 
 Before programming the Pi to make the plant watering robot functional, you need to physically set up the plant watering robot by wiring the different components together. You will set up the robot to receive signals from the resistive soil moisture sensor and signal to the pump when it is time to pump water from the water’s container to the plant’s container.
 
 Refer back to this full wiring diagram as you complete the steps to wire your hardware.
 
-![Full Wiring Diagram](assets/DIAGRAM.png)
+![Full Wiring Diagram](assets/plantWateringDiagram.png)
 
-For wiring the sensor module, follow the below chart (see <https://pinout.xyz/> for a detailed look at the Raspberry Pi pin layout):
+> aside positive
+> The website [pinout.xyz](https://pinout.xyz/) is also a helpful resource where you can interact with the exact layout and role of each pin for Raspberry Pi. When working with Viam, make sure to reference the physical pin numbers, and not the GPIO numbers listed on `pinout.xyz`. 
 
-- PIN 4 (3V) to VCC (Power)
+First, wire the **sensor module** to the Raspberry Pi:
+
+![sensor module wiring diagram](assets/sensorModuleWireDiagram.png)
+
+- PIN 4 (5V) to VCC (Power)
```

Comment:
Should we update these directions to use the new table layout format for clarity?